pages/connections/farmlands.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <header
-      class="not-prose bg-bottom bg-cover bg-[#174B33] bg-blend-overlay bg-[url('~/assets/images/countryside.jpg')]"
+      class="not-prose bg-bottom bg-cover bg-[#174B33] bg-blend-overlay bg-[url('/countryside.jpg')]"
```

Comment:
@Ray2126 was there a reason for moving this?
I think @tom-blake had a preference for this to be in the assets folder so that it can be processed by webpack to reduce the size of the content being served.

Reply:
Yeah I couldn't get the image to show up in the link preview when it was in the `assets` folder. I could duplicate the image to both folders?

Reply:
I think public is ok but check with Tom 👍

Reply:
Checked with Tom and he is ok with public 👍